client/src/components/Public/Map/Map.js

```diff
@@ -36,19 +47,58 @@ type Props = {
   onChange: (options: MapOptions) => any,
   history: RouterHistory,
 }
-
+const isInSlovakia = (center: [number, number]): boolean => {
+  return (center[0] > SLOVAKIA_BOUNDS[0][1]) && (center[0] < SLOVAKIA_BOUNDS[1][1]) &&
+    (center[1] > SLOVAKIA_BOUNDS[0][0]) && (center[1] < SLOVAKIA_BOUNDS[1][0])
+}
 // NOTE: there can be multiple points on the map on the same location...
-const Map = ({zoom, center, clusters, onChange}: Props) => {
+const Map = ({useLabels, zoom, center, clusters, onChange}: Props) => {
+  let finalClusters = clusters
```

Comment:
I don't understand why `clusters` are here. If I get that right, it can be removed completely, can't it?

Reply:
no, because when you load actual data, it is clustered and in the cluster prop

Reply:
Oh, for some reason, I've read the chain of `if..else` wrong, sorry about that. However, I am slightly dissapointed that we need to load clusters through selector even if we are not using them in the end (if `useLabels` is true). We could move this logic of computing `finalClusters` to the said selector (and make it dependent on `(state, { useLabels })`. What do you think? Feel free to discuss.

Reply:
Logically it would be probably cleaner in the selector, i could actually make a selector for useLabels, one does not need to introduce a separate state property, since it is computed, i will do it that way...